lxd/storage/drivers/driver_dir_utils.go

```diff
@@ -12,6 +12,16 @@ import (
 	"github.com/lxc/lxd/shared/units"
 )
 
+// withoutGetVolID returns a copy of this struct but without the volIdFunc property set.
```

Comment:
@stgraber this description isn't strictly accurate anymore, perhaps it should just say:

```
//withoutGetVolID returns a copy of this struct populated with a volIDFunc that will cause any quota operations to be skipped.
```

Reply:
fixed